### DISCUSSION TITLE
Can't set up Xvfb screen resolution

### DISCUSSION THREAD

**Discussion by serg-sno** [seleniumbase/SeleniumBase#3137] [Q&A] [answered]
Hello!

I run my application on an Ubuntu Server in Oracle VM VirtualBox. How can I change the display resolution? Currently, it is always set to 1366x768.
![_1_screenshot](https://github.com/user-attachments/assets/45450848-5332-499d-9d93-1cc913cf17ed)
To initialize the driver, I use:
`with SB(uc=True, xvfb=True, user_data_dir=os.path.join("chrome_cache", "cache")) as sb:`
I've tried configuring the Xvfb screen before running the application like this:
https://github.com/seleniumbase/SeleniumBase/blob/master/integrations/linux/Xvfb_launcher.sh
but it didn’t help.

Could you please tell me what I’m doing wrong?

**Comment by mdmintz** [comment, marked as answer]
Duplicate of https://github.com/seleniumbase/SeleniumBase/discussions/3101#discussioncomment-10571812 and https://github.com/seleniumbase/SeleniumBase/discussions/3101#discussioncomment-10577814.
